config/csm/test_rest_iam_user.yaml

```diff
@@ -1,6 +1,7 @@
 common:
```

Comment:
We want to have flexibility to run each test with different parameter if we want. This should be part of test config. If at all there is need for common config, it should be moved to csm_config.yaml

Reply:
okay